### PR TITLE
docs(dsystemtime): add docs for dsystemtime

### PR DESCRIPTION
**Pull request by xzl01**
添加文档

Log: 添加文档

### CODE REVIEW

**Comment by toberyan** [comment]
+1

File: dtksystemtime/include/dsystemtime.h [hunk 1]
```diff
@@ -64,7 +64,7 @@ class DSystemTime : public QObject
 public slots:
     QStringList listTimeZones() const;
     void setLocalRTC(const bool local_rtc, const bool fix_system, const bool interactive);  // TODO: fix system是否可以被丢掉
-    void setNTP(const bool use_NTP, const bool interactive);  // FIXME: this functions should be fixing
+    void setNTP(const bool use_NTP, const bool interactive);                                //
```
Comment (Groveer) [inline dtksystemtime/include/dsystemtime.h]: 空注释

**Review by Groveer — COMMENTED** [review]
+1

**Review by dandan-y — COMMENTED** [review]
+1

**Review by Groveer — APPROVED** [review]

**Comment by uniontechzhangsong** [comment]
+1

**Pull request merged**